xc/common/utils/prjxray_create_place_constraints.py

```diff
@@ -741,6 +757,63 @@ def main():
                 block, vpr_loc, "Constraining clock block {}".format(block)
             )
 
+    """ Constrain IDELAYCTRL sites
+
+    Prior to the invocation of this script, the IDELAYCTRL sites must have been
+    replicated accordingly to the IDELAY specifications.
+    There can be three different usage combinations of IDELAYCTRL and IDELAYs in a design:
+        1. IODELAYs and IDELAYCTRLs can be constrained to banks as needed.
```

Comment:
#1 should be clarified as a "LOC" constraint, as that is what we mean.

Reply:
Done